diff -urpN initscripts-9.31/systemd/fedora-storage-init initscripts-9.31-patched/systemd/fedora-storage-init
--- initscripts-9.31/systemd/fedora-storage-init	2011-06-21 14:41:29.000000000 -0500
+++ initscripts-9.31-patched/systemd/fedora-storage-init	2011-07-19 11:44:30.421998951 -0500
@@ -9,7 +9,7 @@
 if ! strstr "$cmdline" nompath && [ -f /etc/multipath.conf ] && \
 		[ -x /sbin/multipath ]; then
 	modprobe dm-multipath > /dev/null 2>&1
-	/sbin/multipath -v 0
+	/sbin/multipath  -n -v 0
 	if [ -x /sbin/kpartx ]; then
 		/sbin/dmsetup ls --target multipath --exec "/sbin/kpartx -a -p p" >/dev/null
 	fi
